styles/styles.css

```diff
@@ -109,7 +107,7 @@ body {
   line-height: 1.6;
   color: var(--body-text-color);
   background-color: var(--background-color);
-  background-image: linear-gradient(to bottom, var(--background-color), var(--background-color-gradient)); 
+  background-image: linear-gradient(to bottom, var(--background-color)); 
```

Comment:
We do not need background-image property as background-color is already set.

Reply:
done